jwst/regtest/test_miri_dark.py

```diff
@@ -1,16 +1,14 @@
 import pytest
 from astropy.io.fits.diff import FITSDiff
 
-from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
 from jwst.stpipe import Step
 
 
 @pytest.mark.bigdata
 def test_miri_dark_pipeline(_jail, rtdata, fitsdiff_default_kwargs):
     rtdata.get_data("miri/image/jw00001001001_01101_00001_mirimage_uncal.fits")
 
-    collect_pipeline_cfgs("config")
-    args = ["config/calwebb_dark.cfg", rtdata.input]
+    args = ["jwst.pipeline.DarkPipeline", rtdata.input]
```

Comment:
One could also use the alias `calwebb_dark` here, and on the ones below, instead of the fully-qualified class name.